submissions/TaniaBondarenko/a-tiny-js-world-update/index.js

```diff
@@ -0,0 +1,73 @@
+class Inhabitant {
+  constructor(name, gender, saying, friends,legs) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+  }
+  toString() {
+    return (` ${this.legs} legs. I\'m <b>${this.name}</b>, a ${this.gender}. To my friend ${this.friends} I always say: "${this.saying}"`);
+  }
+};
+
+class Person extends Inhabitant {
+  constructor(name, gender, saying, friends, species='human', legs=2, hands=2) {
+    super(name, gender, saying, friends,legs)
+    this.species = species;
+    this.hands = hands;
+  }
+  toString() {
+    return `I'm a <em>${this.species}</em> and I have ${this.hands} hands and` + super.toString();
+  } 
+};
+
+class Animal extends Inhabitant {
+  constructor(name, gender, saying, species, friends, legs=4) {
+    super(name, gender, saying, friends,legs )
+    this.species = species;
+  }
+  toString() {
+    return `Since I'm a <em>${this.species}</em> I have only` + super.toString();
+  } 
+};
+
+class Creature extends Person{
+  constructor(name, gender, saying, friends, species = 'half-human', legs = 2, hands = 2) {
+    super (name, gender, saying, friends, species, legs=2, hands=2) 
+  }
+  toString() {
+    return super.toString();
+  } 
```

Comment:
A child class inhertis all properties and methods from its ancestors.
If we do not need to modify an inherited method behaviour then there is no need reassigning it.

Reply:
Got it! Thank you.